api/v1beta1/awscluster_webhook.go

```diff
@@ -164,6 +165,7 @@ func (r *AWSCluster) ValidateUpdate(old runtime.Object) error {
 
 	allErrs = append(allErrs, r.Spec.Bastion.Validate()...)
 	allErrs = append(allErrs, r.Spec.AdditionalTags.Validate()...)
+	allErrs = append(allErrs, r.Spec.S3Bucket.Validate()...)
```

Comment:
I think it's completely fine to create a bucket for existing clusters, e.g. when you want to migrate the existing cluster to Flatcar or when you want to add Flatcar worker pool.

Reply:
I got this idea because of this comment:


```
// Create controls if S3 Bucket should be created as part of cluster provisioning.
	// +optional
	Create bool `json:"create,omitempty"`
```

Might wanna change it.

Reply:
The create field is now gone, so I'm going to mark it as resolved then. Let me know if you think there is still something to do regarding the update validation logic.